components/routing.rst

```diff
@@ -346,6 +346,181 @@ automatically in the background if you want to use it. A basic example of the
     are saved in the ``cache_dir``. This means your script must have write
     permissions for that location.
 
+Unicode Routing support
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The Routing component supports UTF-8 characters in route paths and requirements.
+Thanks to the ``utf8`` route option, you can make Symfony match and generate
+routes with UTF-8 characters:
+
+.. configuration-block::
+
+  .. code-block:: php-annotations
+
+        // src/AppBundle/Controller/DefaultController.php
+        namespace AppBundle\Controller;
+
+        use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+        use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+
+        class DefaultController extends Controller
+        {
+            /**
+             *
+             * @Route("/category/{name}", name="route1", options={"utf8": true})
+             *
+             */
+            public function categoryAction()
+            {
+                // ...
+            }
+
+  .. code-block:: yaml
+
+      # routes.yml
+      route1:
+          path:     /category/{name}
+          defaults: { _controller: 'DefaultController::categoryAction' }
+          options:
+            utf8: true
+
+  .. code-block:: xml
+
+        <!-- app/config/routing.xml -->
+        <?xml version="1.0" encoding="UTF-8" ?>
+        <routes xmlns="http://symfony.com/schema/routing"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="http://symfony.com/schema/routing
+                http://symfony.com/schema/routing/routing-1.0.xsd">
+
+            <route id="route1" path="/category/{name}">
+                <default key="_controller">AppBundle:Default:category</default>
+                <option key="utf8">true</option>
```

Comment:
Question here: how can we do to ensure true will be converted to boolean value ?